StateWrappers.as

```diff
@@ -1,52 +1,99 @@
 #if MP4
 
+const uint WheelsStartOffset = 0x53C;
```

Comment:
It's worth noting that these offsets are in `CSceneVehicleVis`, not `CSceneVehicleVisState` (which is actually a part of `CSceneVehicleVis`).